### PR TITLE
Feature/revision requests

### PR DESCRIPTION
**Pull request by FelixSFD**
I have added `SugestedEdit` and `Revision` including all related requests

### CODE REVIEW

File: Sources/RequestsRevisions.swift [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            } catch {
+                completionHandler(nil, error)
+            }
+        }
+    }
+    
+    /**
+     Fetches a revision synchronously.
+     
+     - parameter ids: The revision ID to fetch.
```
Comment (jnkel) [inline Sources/RequestsRevisions.swift]: `ids` should be `id` here and on line 113

File: Sources/RequestsSuggestedEdits.swift [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            } catch {
+                completionHandler(nil, error)
+            }
+        }
+    }
+    
+    
+    /**
+     Fetches a `SuggestedEdit` synchronously.
+
```
Comment (jnkel) [inline Sources/RequestsSuggestedEdits.swift:143]: `ids` again.  I might have gotten this wrong in RequestsQuestions.

**Review by NobodyNada — CHANGES_REQUESTED** [review]
Great!  Just a couple of little documentation things.  Merry Christmas!

**Comment by FelixSFD** [comment]
I fixed the parts you mentioned. There are some more problems (documentation of completion handlers), but let's fix them in an other branch

**Review by NobodyNada — APPROVED** [review]

**Pull request merged**